lib/local/version.js

```diff
@@ -18,7 +28,7 @@ module.exports = function(browser) {
 
     debug('Retrieving version for windows executable', command);
     // Can't use Q.nfcall here unfortunately because of non 0 exit code
-    exec(command, function(error, stdout) {
+    exec(command.split(' ')[0], command.split(' ').slice(1), function(error, stdout) {
```

Comment:
Can you please change this `exec()` to `execFile()`

Reply:
@Mik317 - once you have made the fix - we will get this merged - cheers! 🍰

Reply:
Ok, I just updated the fix as requested. In the meanwhile I also applied a `minor fix` because I had inserted certain characters as `blacklisted` that however could be part of a `path` (not only of a `filename`) :).

Regards,
Mik

Reply:
@Mik317 - awesome, thanks for making the changes!